js/plates-model/model-test-helpers.ts

```diff
@@ -11,8 +11,16 @@ export function getFieldElevation(model: Model, plateId: number, fieldId: number
   return getField(model, plateId, fieldId)?.elevation;
 }
 
-export function isFieldSubducting(model: Model, plateId: number, fieldId: number) {
-  return getField(model, plateId, fieldId)?.subduction?.progress || 0 > 0;
+export function isFieldUnderneathSubducting(model: Model, plateId: number, fieldId: number) {
+  return getField(model, plateId, fieldId)?.subductingFieldUnderneath?.subduction?.progress || 0 > 0;
```

Comment:
I think that's not working fully as intended. I guess you meant:
`(getField(model, plateId, fieldId)?.subductingFieldUnderneath?.subduction?.progress || 0) > 0;`
Since you're comparing with 0, there's a small difference in practice (you'd usually return undefined or a number instead of boolean), but still.